src/sdk/namespace_fs.js

```diff
@@ -2372,6 +2372,8 @@ class NamespaceFS {
         }
     }
 
+    // TODO - improve this function so it will be mutual to namespace_fs and bucketspace_fs
+    // (like in accountspace_fs added argument for entity)
```

Comment:
@shirady sounds like this function exists in namespace_fs, bucketspace_fs, and accountspace_fs, can you move accountspace_fs._translate_error_codes() to a native_fs_utils.js and call it with entity from bucketspace_fs and namespace_fs?

Reply:
I thought doing it as a part of fixing the issue would be hard for backport, hence I added this comment and hoped to do it in the future, but since you suggested I changed the code to what I planned in this PR.

Reply:
BTW, what is the difference between the utils in `native_fs_utils` and `fs_utils`?